danger-klaxit/lib/danger_plugin.rb

```diff
@@ -16,13 +16,55 @@ class Danger::DangerKlaxit < Danger::Plugin
 
   def common
     fail_for_bad_commits
+    warn_for_pg_between
     warn_for_public_methods_without_specs
     warn_for_bad_order_in_config
     warn_rubocop
     fail_for_not_updated_structure_sql
     run_brakeman_scanner if rails_like_project?
   end
 
+  # Warn for PostgreSQL BETWEEN usage
+  # Only for Ruby files and non commented lines
+  def warn_for_pg_between
+    git.modified_files.each do |file|
+      next unless file.end_with?(".rb")
+
+      line_number = nil
+      git.diff_for_file(file).patch.split("\n").each do |line|
+        # Header of diff @@ +start,count -start,count @@
+        if line.match?(/^@@\s-\d+,{1}\d+\s\+\d+,\d+\s@@.*$/)
+          line_number = line[/\+[{1}\d+]*,/][/\d+/].to_i - 2
```

Comment:
J'ai un peu de mal à comprendre cette ligne, tu as un exemple?

Reply:
C’est pour récupérer le numéro de la ligne de départ dans le diff

Par exemple
```
irb(main):001:0> "@@ -217,7 +218,7 @@"[/\+[{1}\d+]*,/][/\d+/]
=> "218"
```

Reply:
Okay, but why subtract 2 to the line number?